### DISCUSSION TITLE
I can't figure out how to add a window into a tabbed group?

### DISCUSSION THREAD

**Discussion by nearlyNonexistent** [i3/i3#5759] [Q&A] [answered]
I can make a tabbed group, and I can add multiple entries to it when creating it, but I can't figure out how to add one to an existing tabbed group? I tried dragging it in at all the different highlight points, I tried googling, searching Reddit, and this repository. I feel like a bit of an idiot, lol.

**Comment by orestisfl** [comment, marked as answer]
You can just move windows around with $mod+Shift+$left/$right/$up/$down. By default these are set to:
```
set $up l
set $down k
set $left j
set $right semicolon
```

If you want to drag and drop a window onto a tiling container, the tiling container needs to be visible and you need to drop it on the center of the target container.

**Reply by nearlyNonexistent** [reply]
No idea why it wasn't working before, but I try dragging it in that way and it works now.
